app/modules/key_management/secret_manager.py

```
@@ -20,6 +20,10 @@
     IntegrationKey,
     CreateIntegrationKeyRequest,
     UpdateIntegrationKeyRequest,
+    WorkflowSecret,
```

Ruff (0.11.9)

`app.modules.key_management.secrets_schema.WorkflowSecret` imported but unused

Remove unused import: `app.modules.key_management.secrets_schema.WorkflowSecret`

(F401)